src/Sulu/Bundle/ResourceBundle/DataFixtures/operators.xml

```diff
@@ -88,18 +110,24 @@
             <translation locale="en">is not</translation>
         </translations>
     </operator>
-    <operator type="date" inputType="select" operator="between">
+    <operator type="date" inputType="select" operator="within">
```

Comment:
refactored the operators in the next PR (https://github.com/sulu-io/sulu/pull/1103) where I needed them to build the additional queries